subjects/notdecimal/README.md

```diff
@@ -0,0 +1,53 @@
+## notdecimal
+
+### Instructions
+
+Write a function called `NotDecimal()` that takes as an argument a `string` in forme of a float number with the decimal point and returns a string converted to an `int` without the decimal point (you will have to multiply it by 10^n to remove the `.`).
+
+- If the number doesn't have a decimal point or there is only a zero after the `.` return the number followed by a newline `\n`.
+- If the argument is empty return newline `\n`.
```

Comment:
fix: If the argument is empty return a newline `\n`.

Reply:
It is fixed.